pom.xml

```diff
@@ -118,7 +118,17 @@ SOFTWARE.
       <dependency>
         <groupId>com.artipie</groupId>
         <artifactId>asto</artifactId>
-        <version>0.27</version>
+        <version>0.28</version>
+      </dependency>
+      <dependency>
+        <groupId>com.artipie</groupId>
+        <artifactId>http</artifactId>
```

Comment:
@olegmoz hm, are you sure this will work? All our repositories (asto including) use ppom and http has asto as a dependency

Reply:
@olenagerasimova `dependencyManagement` section just specifies the versions and does not add dependencies automatically. This works for `asto` now - the module does not get included into itself.